modules/ossm-smcp-prod.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deploy-production.adoc
+
+[id="ossm-smcp-prod_{context}"]
+= Configuring your ServiceMeshControlPlane resource for production
+
+If you have installed a basic `ServiceMeshControlPlane` resource to test {ProductShortName}, you must configure it to production specifications before you deploy your service mesh in production.
+
+You cannot change the `metadata.name` field of an existing `ServiceMeshControlPlane` resource. For production deployments you _must_ customize the default template. 
```

Comment:
IBM SG on italics, 158-163. Though not incorrect to italicize `must` the style guide doesn't give much uses of italicizing for dramatic purposes. I also don't see italics to direct attention like this very much. 

Do you think `must customize` is strong enough language to communicate to users? That is, do you think the italics are necessary? 

If they are necessary, consider a note or warning to communicate to users the importance of customizing their default template.